### PR TITLE
Feature/room crud

### PR DESCRIPTION
**Pull request by facamesi52**

### CODE REVIEW

**Review by viamazoba — APPROVED** [review]

File: src/pages/AdminRoomEdit/index.jsx [hunk 1]
```diff
@@ -33,46 +58,47 @@ const AdminRoomEdit = () => {
...
-                  urlImage={room.img}
-                  arrayAmenities={room.amenities}
-                  arrayInclusions={room.inclusion}
-                  previousPrice={room.previousPrice}
-                  newPrice={room.newPrice}
+                  roomId={room.id}
+                  roomName={room.room_name}
+                  urlImage={room.room_img}
+                  // arrayAmenities={room.amenities}
+                  // arrayInclusions={room.inclusion}
```
Comment (ht1204) [inline src/pages/AdminRoomEdit/index.jsx:88]: Delete commented code

File: src/pages/AdminRoomEdit/index.jsx [hunk 2]
```diff
@@ -33,46 +58,47 @@ const AdminRoomEdit = () => {
...
-                  dateIn={room.dateIn}
-                  dateOut={room.dateOut}
+                  // country={room.country}
+                  // city={room.city}
+                  hotelName={room.hotel_name}
+                  guests={room.max_guests}
+                  // checkIn={room.checkIn}
+                  // checkOut={room.checkOut}
+                  // dateIn={room.dateIn}
+                  // dateOut={room.dateOut}
```
Comment (ht1204) [inline src/pages/AdminRoomEdit/index.jsx:99]: Same here

File: src/pages/FormRoomEdit/index.jsx [hunk 3]
```diff
@@ -1,118 +1,89 @@
...
+  //   'Couch',
+  //   'Shower',
+  // ];
+  // const availableInclusions = [
+  //   'Wi-Fi',
+  //   'Breakfast',
+  //   'Free Cancellation',
+  //   'Non Refundable',
+  //   'Dinner & Lunch',
+  // ];
```
Comment (ht1204) [inline src/pages/FormRoomEdit/index.jsx:39]: AND here

File: src/router/index.jsx [hunk 4]
```diff
@@ -112,17 +112,17 @@ const router = createBrowserRouter([
       {
         path: '/admin-room-edit',
         element: (
-          <PrivateRoute role="admin" redirectRoute="/login">
-            <AdminRoomEdit />
-          </PrivateRoute>
+        // <PrivateRoute role="admin" redirectRoute="/login">
```
Comment (ht1204) [inline src/router/index.jsx:115]: AND here

File: src/store/AppContext.jsx [hunk 5]
```diff
@@ -131,6 +148,41 @@ export const AppContextProvider = ({ children }) => {
...
     }
   };
+  // const openModal = (hotel) => {
+  //   setSelectedHotelForModal(hotel);
+  //   setShowModal(true);
+  // };
+  // const closeModal = () => {
+  //   setSelectedHotelForModal(null);
+  //   setShowModal(false);
+  // };
```
Comment (ht1204) [inline src/store/AppContext.jsx:158]: AND here

**Review by ht1204 — APPROVED** [review]
Approve, but delete commented code before merge

**Pull request merged**